src/watchmaker/static/config.yaml

```diff
@@ -1,4 +1,4 @@
-watchmaker_version: ">= 0.27.2.dev"
+watchmaker_version: ">= 0.27.6.dev"
```

Comment:
Based on what we know at the moment, this doesn't need a change.

We'd only update the min version here if testing of salt 3006.2 reveals an error that we have to fix in a salt formula, such that the fix would only be packaged in the new version of watchmaker...